tests/unit/test_finetune_resources.py

```diff
@@ -31,6 +35,41 @@
 )
 
 
+def mock_request(options: TogetherRequest, *args, **kwargs):
+    if options.url == "fine-tunes/estimate-price":
+        return (
+            TogetherResponse(
+                data={
+                    "estimated_total_price": 100,
+                    "allowed_to_proceed": True,
+                    "estimated_train_token_count": 1000,
+                    "estimated_eval_token_count": 100,
+                    "user_limit": 1000,
+                },
+                headers={},
+            ),
+            None,
+            None,
+        )
+    elif options.url == "fine-tunes":
+        return (
+            TogetherResponse(
+                data={
+                    "id": "ft-12345678-1234-1234-1234-1234567890ab",
+                },
+                headers={},
+            ),
+            None,
+            None,
+        )
+    else:
+        return (
```

Comment:
Better to explicitly check that it's equal to /limits (since this is what we return here) and raise ValueError if none of the options would match. A dispatch table pattern is a good option here